docs/sponsors.md

```
@@ -0,0 +1,80 @@
+These companies support the en.dev open source project family.
+
+<div v-if="error" class="sponsors-note">
+  Sponsor data could not be loaded. Visit <a href="https://en.dev/sponsor.html">en.dev sponsors</a>.
+</div>
+<div v-else-if="!feed" class="sponsors-note">Loading sponsors...</div>
+<div v-else>
+  <section v-for="tier in sponsorsByTier" :key="tier.id" class="sponsor-tier">
+    <h2>{{ tier.label }}</h2>
+    <div v-if="tier.sponsors.length" class="sponsor-grid">
+      <a v-for="sponsor in tier.sponsors" :key="sponsor.name" class="sponsor-card" :href="sponsor.url" target="_blank" rel="noopener noreferrer">
+        <img v-if="sponsor.logo" :src="sponsor.logo" :alt="sponsor.name">
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)